content/6_Advanced/parallel_binary_search.mdx

```diff
@@ -0,0 +1,223 @@
+---
+id: parallel-binary-search
+title: 'Parallel Binary Search'
+author: Talha Taki
+prerequisites:
+  - binary-search
+description: It's an amazing technique for doing the binary search for all queries parallely.
+frequency: 0
+---
+
+<FocusProblem problem="sample" />
+
+<Resources>
+	<Resource
+		source="Robert's Blog"
+		title="A SImple Introduction to Parallel Binary Search"
+		url="https://robert1003.github.io/2020/02/05/parallel-binary-search.html"
+		starred
+	>
+	</Resource>
+		<Resource
+		source="CF"
+		title="Tutorial on Parallel Binary Search"
+		url="https://codeforces.com/blog/entry/45578"
+		starred
+	>
+	</Resource>
+
+</Resources>
+Both of the above resources provide a solution to [Meteors](https://szkopul.edu.pl/problemset/problem/7JrCYZ7LhEK4nBR5zbAXpcmM/site/?key=statement) using **parallel binary search** technique.
+
+<Info>
+This section is based on the definitions given in [Binary Search](/silver/binary-search) module.
+</Info>
+## Tutorial
+
+<IncompleteSection>
```

Comment:
you can use a self-closing tag here (`<IncompleteSection />`). same applies to the resource tags above.

Reply:
changed it